test/test_runner.rb

```diff
@@ -1,35 +0,0 @@
-require 'ci/reporter/minitest'
-
-module ForemanMiniTestRunner
```

Comment:
At the very least then we'll need to print a warning in the plugin registration method when people use the skip method if you don't plan to replace it in this PR.

I notice our test for the skip runner in test/lib/custom_runner.rb is passing, I think probably because it's misnamed and should have a _test extension.  Could you please rename that and add a "skip" to the file so we know why the er, skip test is skipped?